bitnami/kafka/templates/jmx-configmap.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: {{ printf "%s-jmx-configuration" (include "common.names.fullname" .) }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: metrics
```

Comment:
Shouldn't this be cluster-metrics too?

Reply:
i put `metrics` for jmx (sidecar/1 per broker) & `cluster-metrics` for kafka exporter (1 deployment per cluster)

Reply:
I put it there for completeness (some all *jmx*.yaml have it), but I can revert it too

Reply:
Fair enough